### DISCUSSION TITLE
UI regarding the recycle bin

### DISCUSSION THREAD

**Discussion by gera2ld** [violentmonkey/violentmonkey#1714] [Polls]
Recently we have moved the recycle bin to a new tab, of which the entrance is at the bottom of the side menu because we don't want it to take too much of your attention.
But it seems some people have different opinions regarding the UI.

**Comment by tophf** [comment]
At the bottom of the page + a big trashcan icon

![image](https://user-images.githubusercontent.com/1310400/211220885-f79e0cd9-a2db-4c3e-ad99-0cd39cba0a7a.png)

**Comment by mikelei8291** [comment]
I think since the removed scripts list is on a whole page that has the same UI level and position with other whole page UI elements (installed scripts list, settings page, and about page), the entrance of it should also be at the same level of their entrances. Therefore, I think it should be under the "Installed Scripts" menu item.

The previous implementation (inside the "Installed Scripts" page) would lower its visibility and always makes me wonder where it is after deleting a script in the list. However, placing it at the bottom of the page also makes it overemphasized (not to mention adding a big icon above it), cause it is not something that need extra attention from the user.

**Reply by gera2ld** [reply]
I thought it could draw less attention by putting the entrance in a place far from the center.
Would it be too noisy to have a tab that is less frequently used in the main area?

**Reply by mikelei8291** [reply]
I'm not a UI designer by any means, but I would recommend you to look at existing examples:

1. Google Drive
   ![Google Drive](https://user-images.githubusercontent.com/9620335/212371132-18cb14b5-f5fc-4398-996f-8815377d597f.png)
2. Files app on iPadOS
   ![Files app on iPadOS](https://user-images.githubusercontent.com/9620335/212371503-7f212745-d89c-43a4-95b4-bbaffceebb6b.png)

**Reply by mikelei8291** [reply]
In my opinion, "less frequent" doesn't usually mean "less important". Also, I think users would assume that the entrances of pages at the same UI level would be placed together, instead of all around the webpage.
